File: Cenfomon/Cenfomon_COI/bl/entities/Message.ts
import { Answer } from "./Answer";
import {NPC} from "./NPC";

export class Message{
    private _id : number;
    private _to : NPC;
    private _from:NPC;
    private _reply: Answer;


    constructor(id: number, to: NPC, from: NPC, reply: Answer) {
        this._id = id;
        this._to = to;
        this._from = from;
        this._reply = reply;
    }


    getid(): number {
        return this._id;
    }

    setid(value: number): void {
        this._id = value;
    }

    get to(): NPC {
        return this._to;
    }

    set to(value: NPC) {
        this._to = value;
    }

    get from(): NPC {
        return this._from;
    }

    set from(value: NPC) {
        this._from = value;
    }

    get reply(): Answer {
        return this._reply;
    }

    set reply(value: Answer) {
        this._reply = value;
    }
}

File: Cenfomon/Capa_Logica/src/main/java/com/cenfomon/Cenfomon_Capa_Logica/bl/entities/Message.java
package com.cenfomon.Cenfomon_Capa_Logica.bl.entities;

public class Message {
    /**
     * Atributos
     */
    int id;
    NPC to;
    NPC from;
    Answer reply;
    /**
     * get and set
     */
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public NPC getTo() {
        return to;
    }

    public void setTo(NPC to) {
        this.to = to;
    }

    public NPC getFrom() {
        return from;
    }

    public void setFrom(NPC from) {
        this.from = from;
    }

    public Answer getReply() {
        return reply;
    }

    public void setReply(Answer reply) {
        this.reply = reply;
    }
}
